app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,77 @@
+class TweetsController < ApplicationController
+   
+   get '/tweets' do 
+    if is_logged_in?
+    
```

Comment:
@johnsontaiwo why is there not an instance variable here for `@tweets = Tweet.all` ?